docs/oasis-network-primer/staking_rewards/data.csv

```diff
@@ -0,0 +1,446 @@
+Until epoch,Estimated date,Reward per epoch %,Estimated annualized rewards %
+1170,2020-11-18T16:00:00.000Z,0.002081,20.0697
+4842,2021-04-20T03:46:38.848Z,0.002081,20.0697
+4866,2021-04-21T03:41:51.259Z,0.002080,20.0592
```

Comment:
Instead of just making CSV from genesis:
```
Until,Scale
1170,2081
4842,2081
4866,2080
```

Using https://github.com/oasisprotocol/oasis-core/blob/d8e352b/go/staking/api/rewards.go#L22 (rewardAmountDenominator = 100_000_000) I parsed the scale as `/100_000_000 * 100%`  and added estimations to CSV too.

```js
      const estimatedEpochsPerYear = 365 * 24 * estimatedEpochsPerHour;
      const rewardPerEpoch = scale / rewardAmountDenominator;
      const yearlyCompoundedRate = (1 + rewardPerEpoch)**estimatedEpochsPerYear
```

Estimated with estimatedEpochsPerHour: 1
```
Until epoch,Estimated date,Reward per epoch %,Estimated annualized rewards %
1170,2020-11-16T15:11:11.000Z,0.002081,19.9967
4842,2021-04-18T15:11:11.000Z,0.002081,19.9967
4866,2021-04-19T15:11:11.000Z,0.002080,19.9861
```

Estimated with estimatedEpochsPerHour: 1.003339695445333
```
Until epoch,Estimated date,Reward per epoch %,Estimated annualized rewards %
1170,2020-11-18T16:00:00.000Z,0.002081,20.0697
4842,2021-04-20T03:46:38.848Z,0.002081,20.0697
4866,2021-04-21T03:41:51.259Z,0.002080,20.0592
```